lighthouse-core/config/default-config.js

```diff
@@ -424,6 +424,9 @@ const defaultConfig = {
     'best-practices-general': {
       title: str_(UIStrings.bestPracticesGeneralGroupTitle),
     },
+    'hidden': {
```

Comment:
Add a comment explaining the purpose of this group? `plugins` and anyone doing custom configs might want to use it too.

Reply:
> `plugins` and anyone doing custom configs might want to use it too.

actually plugins get all their groups prepended with `lighthouse-plugin-pluginname-`, so `hidden` on its own won't work. We'll have to give them a solution another day :)